client/repositories/operation/operation.go

```diff
@@ -0,0 +1,193 @@
+package operation_repo
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/lidofinance/dc4bc/client/modules/state"
+	"github.com/lidofinance/dc4bc/client/types"
+)
+
+const (
+	OperationsKey        = "operations"
+	DeletedOperationsKey = "deleted_operations"
+	FSMStateKey          = "fsm_state"
+)
+
+type OperationRepo interface {
+	PutOperation(operation *types.Operation) error
+	DeleteOperation(operation *types.Operation) error
+	GetOperations() (map[string]*types.Operation, error)
+	GetOperationByID(operationID string) (*types.Operation, error)
+}
+
+type BaseOperationRepo struct {
+	state                        state.State
+	operationsCompositeKey       string
+	deleteOperationsCompositeKey string
+}
+
+func NewOperationRepo(s state.State, topic string) (*BaseOperationRepo, error) {
+	operationsCompositeKey := state.MakeCompositeKeyString(topic, OperationsKey)
+	deleteOperationsCompositeKey := state.MakeCompositeKeyString(topic, DeletedOperationsKey)
+
+	repo := &BaseOperationRepo{
+		state:                        s,
+		operationsCompositeKey:       operationsCompositeKey,
+		deleteOperationsCompositeKey: deleteOperationsCompositeKey,
+	}
+
+	if err := repo.initJsonKey(operationsCompositeKey, make(map[string]*types.Operation)); err != nil {
```

Comment:
is there a need for initJsonKey for operations and deleteOperations with empty maps as the arguments? a marshalled empty map looks like `{}` in json, so, if we need to init it in the state, we can simply run
```golang
r.state.Set(key, []byte("{}"))
```
can't we?